tests/bento-log-redaction.unit.test.ts

```
@@ -0,0 +1,119 @@
+})
+
+afterEach(() => {
+  vi.unstubAllGlobals()
+  vi.restoreAllMocks()
+  cloudlogErrMock.mockReset()
+  cloudlogMock.mockReset()
+  fetchMock.mockReset()
+  getEnvMock.mockClear()
+})
+
+describe('Bento log redaction', () => {
```

ESLint

[error] `describe`s should begin with lowercase

(test/prefer-lowercase-title)